tests/unit/mixins/Projector.ts

```diff
@@ -126,18 +126,96 @@ registerSuite({
+				assert.strictEqual(child.innerHTML, 'foo');
+				assert.strictEqual(child.tagName.toLowerCase(), 'h2');
+				document.body.removeChild(div);
+			},
+			'pre rendered DOM used'() {
+				const iframe = document.createElement('iframe');
+				document.body.appendChild(iframe);
+				iframe.contentDocument.write(`
+					<div class="foo">
+						<label for="baz">Select Me:</label>
+						<select type="text" name="baz" id="baz" disabled="disabled">
+							<option value="foo">label foo</option>
+							<option value="bar" selected="">label bar</option>
+							<option value="baz">label baz</option>
+						</select>
+						<button type="button" disabled="disabled">Click Me!</button>
+					</div>`);
+				iframe.contentDocument.close();
+				const root = iframe.contentDocument.body.firstChild as HTMLElement;
+				const childElementCount = root.childElementCount;
+				const select = root.childNodes[3] as HTMLSelectElement;
+				const button = root.childNodes[5] as HTMLButtonElement;
+				assert.strictEqual((root.childNodes[3] as HTMLSelectElement).value, 'bar', 'bar should be selected');
+				const onchangeListener = spy();
+				const onclickListener = spy();
+				const projector = new class extends TestWidget {
+					render() {
+						return v('div', {
+							classes: { foo: true, bar: true }
+						}, [
+							v('label', {
+								for: 'baz'
+							}, [ 'Select Me:' ]),
+							v('select', {
+								type: 'text',
+								name: 'baz',
+								id: 'baz',
+								disabled: false,
+								onchange: onchangeListener
+							}, [
+								v('option', { value: 'foo', selected: true }, [ 'label foo' ]),
+								v('option', { value: 'bar', selected: false }, [ 'label bar' ]),
+								v('option', { value: 'baz', selected: false }, [ 'label baz' ])
+							]),
+							v('button', {
+								type: 'button',
+								disabled: false,
+								onclick: onclickListener
+							}, [ 'Click Me!' ])
+						]);
+					}
+				}();
+				projector.merge(root);
+				assert.strictEqual(root.className, 'foo bar', 'should have added bar class');
+				assert.strictEqual(root.childElementCount, childElementCount, 'should have the same number of children');
+				assert.strictEqual(select, root.childNodes[3], 'should have been reused');
+				assert.strictEqual(button, root.childNodes[5], 'should have been reused');
+				assert.isFalse(select.disabled, 'select should be enabled');
+				assert.isFalse(button.disabled, 'button shound be enabled');
+
+				assert.strictEqual(select.value, 'foo', 'foo should be selected');
+				assert.strictEqual(select.children.length, 3, 'should have 3 children');
+
+				assert.isFalse(onchangeListener.called, 'onchangeListener should not have been called');
+				assert.isFalse(onclickListener.called, 'onclickListener should not have been called');
+
+				const changeEvent = document.createEvent('Event');
+				changeEvent.initEvent('change', true, true);
+				select.onchange(changeEvent); // firefox doesn't like to dispatch this event
```

Comment:
It isn't a WebDriver defect, it is a difference of implementation of browsers.  Contrary to popular belief, there are inconsistencies in implementation between browsers.  I suspect it has to do with onchange in firefox only occurring when you blur the element (which is different than other browsers) but it could also be firefox's trust issue with generated events.  Both of which weren't material to this test case, so I just moved on and noted what someone might consider an oddity.  If it were material to the test case, I would have expected more detailed information, or if it were a code path in the production code.